components/underlay/index.css

```diff
@@ -13,34 +13,67 @@ governing permissions and limitations under the License.
 @import '../overlay/index.css';
 
 .spectrum-Underlay {
-  /* Bug: this must be 0ms, not 0 */
-  --spectrum-dialog-confirm-background-entry-animation-delay: 0ms;
-  --spectrum-dialog-confirm-background-exit-animation-ease: cubic-bezier(0.5, 0, 1, 1); /* wrong in DNA */
-  --spectrum-dialog-confirm-background-entry-animation-ease: cubic-bezier(0, 0, 0.40, 1); /* wrong in DNA */
+  --spectrum-underlay-background-entry-animation-delay: var(--spectrum-animation-duration-0);   /* Bug: this must be 0ms, not 0 */
```

Comment:
Does this work as-is or do we need to do a hardocoded 0ms for now?

Reply:
It does work as is. I had interrupted this comment as it previously was `0` and it was corrected to be `0ms` then here I am just using our custom animation token for it. 

I couldn't find the related git blame that brought in the comment. Nor did I see any active Github issues for it. I have removed the comment for now as it doesn't seem relevant any more.